lib/mix/tasks/compile.elchemy.ex

```diff
@@ -30,5 +31,8 @@ defmodule Mix.Tasks.Compile.Elchemy do
     Mix.Task.run "deps.get"
     Mix.Task.run "deps.compile"
     IO.puts "-- Elchemy compilation complete --\n"
+
+    if version == "~> 1.7", do: IO.warn "Elixir 1.7 version doesn't work with Elchemy. You have to use lower."
```

Comment:
Also, some smart matching (regex?) that'd exclude everything `1.7.0 <= v < 1.8.0`